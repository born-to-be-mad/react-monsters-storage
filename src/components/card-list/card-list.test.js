import { shallow } from "enzyme";
import React from "react";
import CardList from "./card-list.component";

test("expect to render CardList component - snapshot", () => {
  const mockItems = [
    { id: 101, name: "IVN", username: "Ivan Ivanoff", email: "i.ivanov@tut.by" }
  ];
  const wrapper = shallow(<CardList items={mockItems} />);
  expect(wrapper).toMatchSnapshot();
});
