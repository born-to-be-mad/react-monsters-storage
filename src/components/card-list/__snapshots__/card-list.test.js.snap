// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component - snapshot 1`] = `
<div
  className="card-list"
>
  <Card
    email="i.ivanov@tut.by"
    id={101}
    key="0"
    name="IVN"
  />
</div>
`;
